csharp/ServerSideExtension/ServerSideExtension.csproj

```diff
@@ -40,10 +40,6 @@
       <HintPath>..\packages\Grpc.Core.1.2.2\lib\net45\Grpc.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="NLog, Version=4.0.0.0, Culture=neutral, PublicKeyToken=5120e14c03d0593c, processorArchitecture=MSIL">
```

Comment:
Is this include here not needed anymore? As far as I can see NLog is being used at least..

Reply:
The ServerSideExtension project does not use NLog. It is only the SSEtoRserve project that uses it.
NLog was not part of the packages.config but was referenced in the project file for some reason so I cleaned up that warning intentionally.